src/components/dls/Badge/Badge.tsx

```diff
@@ -0,0 +1,19 @@
+import classNames from 'classnames';
+
+import styles from './Badge.module.scss';
+
+type BadeProps = {
+  content: React.ReactNode;
+  children: React.ReactNode;
+};
+
+const Badge = ({ content, children }: BadeProps) => {
```

Comment:
Right now the bade is very simple, only support position bottom right. 

Do we have other use case? we can also support other colors and other positions